project/src/main/scala/docs/model/Pattern.scala

```diff
@@ -0,0 +1,41 @@
+package docs.model
+
+sealed trait Level
+
+sealed trait Category
+
+object Level {
```

Comment:
Why do you need to duplicate this? you already have this types from codacy-plugins-api in the classpath?